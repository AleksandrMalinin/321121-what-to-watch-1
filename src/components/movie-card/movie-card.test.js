import React from 'react';
import renderer from 'react-test-renderer';
import MovieCard from './movie-card.jsx';

const mock = {
  title: `Fantastic Beasts: The Crimes of Grindelwald`,
  src: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`
};

it(`MainScreen correctly renders`, () => {

  const tree = renderer
  .create(<MovieCard
    movie={mock}
  />)
  .toJSON();

  expect(tree).toMatchSnapshot();
});
