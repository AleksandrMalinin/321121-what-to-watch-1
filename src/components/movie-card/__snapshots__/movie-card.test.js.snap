// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieCard correctly renders 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <a
    href="/film/undefined"
    onClick={[Function]}
  >
    <video
      className="small-movie-card__image"
      height="175"
      muted={true}
      poster="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
      width="280"
    >
      <source
        src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
        type="video/mp4"
      />
    </video>
  </a>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="/film/undefined"
      onClick={[Function]}
    >
      Fantastic Beasts: The Crimes of Grindelwald
    </a>
  </h3>
</article>
`;
